packages/opentelemetry-semantic-conventions-ai/opentelemetry/semconv/ai/__init__.py

```diff
@@ -44,8 +44,17 @@ class SpanAttributes:
     LLM_REPETITION_PENALTY = "llm.watsonx.repetition_penalty"
 
 
-class EventName(Enum):
+class EventAttributes(Enum):
+    # Query Embeddings
     VECTOR_DB_QUERY_EMBEDDINGS = "vector_db.query.embeddings"
```

Comment:
What's the difference between `VECTOR_DB_QUERY_EMBEDDINGS` and `VECTOR_DB_QUERY_EMBEDDINGS_VECTOR`